### PR TITLE
Suppress urllib3 debug log "Converted retries value: False -> Retry"

### PR DESCRIPTION
**Pull request by ikonst**

### CODE REVIEW

**Comment by ikonst** [comment]
See also: https://github.com/elastic/elasticsearch-py/pull/827

**Comment by ikonst** [comment]
Closes #1800.

File: tests/unit/test_http_session.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def assert_request_sent(self, headers=None, body=None, url='/', chunked=False):
             url=url,
             body=body,
             headers=headers,
-            retries=False,
+            retries=ANY,
```
Comment (jamesls) [inline tests/unit/test_http_session.py]: Was there an issue with setting this to `Retry(False)`?

Comment (ikonst) [reply tests/unit/test_http_session.py]: Yes, `Retry` is a class without an `__eq__`.

**Comment by ikonst** [comment]
@jamesls take another look?

**Comment by ikonst** [comment]
@kyleknap @jamesls care to take another look?

**Comment by ikonst** [comment]
@kyleknap @jamesls care to take another look?

**Comment by ikonst** [comment]
@joguSD Do you know who else might have merge permissions?

**Comment by ikonst** [comment]
@joguSD Can you merge this change?

**Comment by joguSD** [comment]
@ikonst Yes, I will merge this today after our release goes out.

**Comment by joguSD** [comment]
I've squashed and merged this in: https://github.com/boto/botocore/commit/38e28378849b61343c0a4faecd9bf17f2eb5ede1

**Pull request closed without merging**